test/testunusedvar.cpp

```diff
@@ -4149,6 +4150,96 @@ class TestUnusedVar : public TestFixture {
         ASSERT_EQUALS("", errout.str());
     }
 
+    void localvarmaybeunused() {
+        std::string passCases[] =
+        {
+R"(
```

Comment:
well I think raw string literals can be nice in some extreme cases if there are lots of special characters. But I prefer to use plain old string literals for these strings.

Reply:
Oh yeah I was going to point this out. I think it's kind of nice but suspected it would be rejected on grounds of consistency if nothing else. Happy to change it.